### ISSUE TITLE
A11y_SQLAzureDataStudio_AddaSQLServerBigDataCluster-Controller_Addnewcontroller_Zoom: After zooming to 400% the  content on the screen is not visible.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9202]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_SQLAzureDataStudioBenchmark;#A11yWCAG2.1;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#Zoom;#MAS1.4.10;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.4.10

**Pre-requisite:**
Set display resolution to 1280x1024, then use DPI zoom to 400%.
(Windows Settings > Display Settings > Scale and Layout)

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Select Add a SQL Server Big Data Clusters.
5. Navigate to + button & hit enter.

**Actual:**
The screen when opened is zoomed to 400% hence the data cannot be seen the labels of the edit box as well as the screen names.

**Issue observed throughout the application except "Add Azure Account ".**

**Expected:**
All data needs to clearly visible on the screen.

**User Impact:**
Users will not be able to see data that is present on the screen and know the purpose of the edit box present.

**Attachment link for Reference :**
[A11y_SQLAzureDataStudio_AddaSQLServerBigDataCluster-Controller_Addnewcontroller_Zoom9202.zip](https://github.com/microsoft/azuredatastudio/files/4322663/A11y_SQLAzureDataStudio_AddaSQLServerBigDataCluster-Controller_Addnewcontroller_Zoom9202.zip)

**Comment by alanrenmsft** [comment]
need to set overflow-x to auto

**Comment by Charles-Gagnon** [comment]
What's the expectation for window size/zoom level that we should support? This isn't a problem unless you get to a extremely small window size (one I would consider to be unreasonably small for a desktop application) and large zoom level.

**Comment by alanrenmsft** [comment]
@Charles-Gagnon what is the status for this bug? we are trying to understand the zooming requirement, right? meanwhile I think if we add the proper overflow, it wouldn't matter what the zooming requirement is.

**Comment by Charles-Gagnon** [comment]
I'm planning on fixing it - just haven't gotten around to it yet.

**Comment by Charles-Gagnon** [comment]
Closing this per guidelines from a11y folks that 2.1 WCAG requirements being pushed to 2021.

**Comment by fsteffi** [comment]
@Charles-Gagnon Reflow Exception link
https://ceacc.visualstudio.com/ceaccex/_workitems/edit/42844

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_Win32;#A11yMAS;

**Comment by Yashi1201** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
**GithubTags**:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
